parlai/scripts/verify_data.py

```diff
@@ -28,7 +28,7 @@ def setup_args(parser=None):
     # Get command line arguments
     parser.add_argument('-ltim', '--log-every-n-secs', type=float, default=2)
     parser.add_argument('-d', '--display-examples', type='bool', default=False)
-    parser.set_defaults(datatype='train:ordered')
+    parser.set_defaults(datatype='train:stream:ordered')
```

Comment:
note @stephenroller we're changing this because otherwise the test_new_tasks test fails on chunk teachers